.docker/ubuntu-22.04/Dockerfile

```diff
@@ -50,6 +64,12 @@ RUN wget -q https://packages.microsoft.com/config/ubuntu/18.04/packages-microsof
         powershell \
   && rm -rf /var/lib/apt/lists/*
 
+RUN wget -q https://github.com/libbpf/bpftool/releases/download/v7.2.0/bpftool-v7.2.0-amd64.tar.gz -O bpftool-amd64.tar.gz && \
```

Comment:
Is bpftool needed for **building** the project?

Reply:
yes, for building ebpf program